### PR TITLE
Fix become a partner button link

### LINKED ISSUE #21: Button "Become a partner" doesn't work
A click on the "Become a partner" button on the homepage doesn't seem to do anything.

### PR DESCRIPTION
**Pull request by KevinTss**
### What this PR does?

Adding a link to button `Become a partner`

@jbelien @colevandersWands I don't know the link, feel free to give it to me here or adding yourself instead of `link here`.
See the comment below

### CODE REVIEW

File: src/components/page-sections/our-partners/OurPartners.jsx [hunk 1]
```diff
@@ -27,7 +27,12 @@ const OurPartners = () => (
     <Wrapper>
       <TopContainer>
         <Title level={2}>{data.title}</Title>
-        <Button variant="minimal" iconRight="arrow-right">
+        <Button
+          is="a"
+          variant="minimal"
+          iconRight="arrow-right"
+          href="link here"
```
Comment (KevinTss) [inline src/components/page-sections/our-partners/OurPartners.jsx]: Should contain the right link here

**Comment by jbelien** [comment]
@colevandersWands @iLienstAr Can you tell us where the "Become a partner" should point to ?

**Review by jbelien — APPROVED** [review]

**Comment by iLienstAr** [comment]
@jbelien @KevinTss it should link to 'contact@hackyourfuture.be', an email window should open, same as in the contact form.

**Review by jbelien — APPROVED** [review]

**Comment by jbelien** [comment]
Thanks 👍

**Pull request merged**